templates/blocks/share_buttons.twig

```diff
@@ -1,23 +1,28 @@
 {% block javascripts %}
     <script>
```

Comment:
This is an idea for another ticket/PR:
IMO, in terms of code separation and maintainability, this JS logic should not be part of the Twig template.
It should exist on a separate file and be included using the `wp_enqueue_script` or similar, which also has benefits in terms of security, caching, minification, etc...

Reply:
I agree! Can you please create a ticket for it? 🙏

Reply:
Done! https://jira.greenpeace.org/browse/PLANET-7625